README.md

```diff
@@ -31,7 +31,7 @@ fastify.register(oauthPlugin, {
 })
 
 fastify.get('/login/facebook/callback', async function (request, reply) {
-  const token = await this.getAccessTokenFromAuthorizationCodeFlow(request)
+  const token = await this.facebookOAuth2.getAccessTokenFromAuthorizationCodeFlow(request)
```

Comment:
We should also update the `## Reference` and `## Utilities` to tell the user that the `name` will be used as key to access those functions

Reply:
I've pushed a change in README.md that adds info about **namespace**. Not sure if it's too descriptive, but feel free to change or cut down details if needed.